Autopilot/AttitudeManager/Peripherals/UHF.c

```diff
@@ -0,0 +1,36 @@
+/* 
+ * @file UHF.h
+ * @author Serj Babayan
+ * @created March 26, 2017, 9:00 PM
+ */
+#include "../InputCapture.h"
+#include "../PWM.h"
+#include "UHF.h"
+
+uint8_t getUHFRSSI(uint32_t sys_time){
```

Comment:
Wasn't actually able to get it to work using the PWM module for whatever reason. Couldn't get values to lie within the 0 - 100 range, probably doing something stupid. Will figure it out later as its low priority.